File: datasets/File04_New.java
// File04_New.java - Line Splitting
public class File04 {
    public void processConfiguration(String host, int port, String username, String password) {
        // The long initialization line is now split
        String connectionString = "jdbc:mysql://" + host + ":" + port + "/";
        connectionString += username + "?password=" + password + "&useSSL=false";
        System.out.println("Connecting to: " + connectionString);
        // More code...
    }
}

File: python-lhdiff-project/datasets/Aakanksha/File04_New.py
# File04_New.py - Line Splitting

class File04:
    def process_configuration(self, host, port, username, password):
        # The long initialization line is now split
        connection_string = f"jdbc:mysql://{host}:{port}/"
        connection_string += f"{username}?password={password}&useSSL=false"

        print("Connecting to:", connection_string)
        print("Connection established.")
        # More code...